### ISSUE TITLE
Rendering dialog doesn't capture 1-6 keys properly

### ISSUE THREAD

**Issue by jahendrie** [jliljebl/flowblade#164]
The problem seems to be that the 'tracks' widget keeps grabbing those key presses even while entering text into a field in a different widget.  I've only tested it in the 'name' field of the rendering widget.

The problem can be worked around with the numeric keypad + numlock, though it's inconvenient.

**Comment by jliljebl** [comment]
Yeah, this can happen some time. Could you give an exact list of actions that is problem for you like the following:

1) Press timeline
2) Press widget X (clear identificaion is needed here)
3) Enter text, buttons 1-6 do not work now

**Comment by jahendrie** [comment]
Alright, I played around a bit with it.  Sequence of events goes something like this:

1)  Open file
2)  Click around at the top of the track to change current position
3)  Click render tab
4)  Double click filename field, enter text, works as it should
5)  Click back in the timeline to change position again, this time pressing 1, 2, etc. to change the tool being used
6)  Click back in filename field (render widget is still displayed at this point), all text entry works as it should
7)  Go back to timeline, change position again, this time clicking on the tool selection (mouse cursor thing) with the mouse
8)  Render widget -> text entry field again, this time 1-6 doesn't work.

I repeated it a couple of times and it seems to be selecting the editing tool with the mouse that does it.
